java/HealthVault Mobile/HVSample/src/main/res/layout/recordpicker.xml

```diff
@@ -0,0 +1,11 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We need to think over our naming convention, here we have recordpicker.xml, but right above is record_picker_item.xml. Lets be consistent.